schema.json

```diff
@@ -202,6 +202,80 @@
             },
             "type": "object"
         },
+        "persist": {
+            "anyOf": [
+                {
+                    "description": "Deprecated, use object representation instead.",
+                    "type": "string"
+                },
+                {
+                    "description": "Deprecated, use object representation instead.",
+                    "items": {
+                        "$ref": "#/definitions/stringOrArrayOfStrings"
```

Comment:
We have #/definitions/stringOrArrayOfStringsOrAnArrayOfArrayOfStrings, why not using it as former?